plugins/tracker-resources/src/components/issues/CategoryPresenter.svelte

```diff
@@ -1,52 +1,97 @@
 <script lang="ts">
-  import { DocumentQuery, FindOptions } from '@anticrm/core'
+  import contact from '@anticrm/contact'
+  import { DocumentQuery, FindOptions, Ref } from '@anticrm/core'
   import { Issue, IssueStatus, Team } from '@anticrm/tracker'
-  import { Scroller } from '@anticrm/ui'
-  import { Table } from '@anticrm/view-resources'
+  import { Icon, IconAdd, Scroller, Tooltip, Button, showPopup, Label } from '@anticrm/ui'
+  import { createEventDispatcher } from 'svelte'
   import tracker from '../../plugin'
-  import contact from '@anticrm/contact'
+  import IssuesList from './IssuesList.svelte'
+  import { issueStatuses } from '../../utils'
+  import CreateIssue from '../CreateIssue.svelte'
 
   export let query: DocumentQuery<Issue>
   export let category: IssueStatus
+  export let currentSpace: Ref<Team> | undefined = undefined
   export let currentTeam: Team
 
+  const dispatch = createEventDispatcher()
+
   const options: FindOptions<Issue> = {
     lookup: {
       assignee: contact.class.Employee
     }
   }
 
-  let visible = false
+  let issuesAmount = 0
+
+  const handleNewIssueAdded = (event: Event) => {
+    if (!currentSpace) {
+      return
+    }
+
+    showPopup(CreateIssue, { space: currentSpace, issueStatus: category }, event.target)
+  }
 </script>
 
-<div class='category' class:visible={visible}>
-  <div class='fs-title'>
-    {IssueStatus[category]}
+<div class="category" class:visible={issuesAmount > 0}>
+  <div class="header categoryHeader flex-between label">
+    <div class="flex-row-center gap-2">
+      <Icon icon={issueStatuses[category].icon} size={'small'} />
+      <span class="lines-limit-2"><Label label={issueStatuses[category].label} /></span>
+      <span class="eLabelCounter ml-2">{issuesAmount}</span>
+    </div>
+    <div class="flex mr-1">
+      <Tooltip label={tracker.string.AddIssueTooltip} direction={'left'}>
+        <Button icon={IconAdd} kind={'transparent'} on:click={handleNewIssueAdded} />
+      </Tooltip>
+    </div>
   </div>
-<Scroller>
-  <Table
-    _class={tracker.class.Issue}
-    config={[
-      { key: '', presenter: tracker.component.IssuePresenter, props: { currentTeam }, label: tracker.string.Issue },
-      'title',
-      // 'status',
-      '$lookup.assignee',
-      'modifiedOn'
-    ]}
-    options={options}
-    query={{ ...query, status: category }}
-    showNotification
-    highlightRows
-    on:content={(evt) => { visible = evt.detail.length > 0 }}
-  />
-</Scroller>
+  <Scroller>
+    <IssuesList
```

Comment:
Probable we should rename IssuesList to ListView and put it inside view/view-resources